src/org/zaproxy/zap/extension/ascanrulesAlpha/resources/Messages.properties

```diff
@@ -50,6 +50,16 @@ ascanalpha.httpsashttpscanner.desc = Content which was initially accessed via HT
 ascanalpha.httpsashttpscanner.soln = Ensure that your web server, application server, load balancer, etc. is configured to only serve such content via HTTPS. Consider implementing HTTP Strict Transport Security.
 ascanalpha.httpsashttpscanner.refs = https://www.owasp.org/index.php/HTTP_Strict_Transport_Security\nhttps://www.owasp.org/index.php/List_of_useful_HTTP_headers\nhttp://en.wikipedia.org/wiki/HTTP_Strict_Transport_Security\nhttp://caniuse.com/stricttransportsecurity\nhttp://tools.ietf.org/html/rfc6797
 
+ascanalpha.integeroverflow.name = Integer Overflow Error
+ascanalpha.integeroverflow.desc = An integer overflow condition exists when an integer, which has not been properly checked as to length is used within a compiled program. 
+ascanalpha.integeroverflow.soln = Rewrite the background program using proper checking of the size of integer being input to prevent overflows and divide by 0 errors.  This will require a recompile of the background executable.
+ascanalpha.integeroverflow.refs = https://www.owasp.org/index.php/Integer_overflow and http://projects.webappsec.org/w/page/13246946/Integer%20Overflows.
+ascanalpha.integeroverflow.error1 = Potential Format String Error.  The script closed the connection on the input of a long string of random integers.
+ascanalpha.integeroverflow.error2 = Potential Format String Error.  The script closed the connection on the input of a long string of zeros.
+ascanalpha.integeroverflow.error3 = Potential Format String Error.  The script closed the connection on the input of a long string of ones.
+ascanalpha.integeroverflow.error4 = Potential Format String Error.  The script closed the connection on the input of a long string of nines.
```

Comment:
The text "Format String Error" should be replaced with "Integer Overflow Error"?
Same comment for the other "error" messages.